src/main/java/uk/gov/ida/matchingserviceadapter/mappers/MatchingDatasetToMatchingDatasetDtoMapper.java

```diff
@@ -114,4 +115,9 @@ public VerifyAddressDto apply(Address input) {
                         input.isVerified()))
                 .collect(Collectors.toList());
     }
+
+    public static Comparator<TransliterableMdsValue> comparatorByVerifiedThenCurrent() {
+        return Comparator.comparing(TransliterableMdsValue::isVerified, Comparator.reverseOrder())
```

Comment:
Why do we need `Comparator.reverseOrder()` when comparing on a boolean value?

Reply:
I assume because the natural ordering of booleans treats `false` as 0, and so less than `true` as 1. In any event, whatever the reason, reverse order ensures that `true`s are sorted first.

Reply:
👍